### DISCUSSION TITLE
Installing Livewire on recently upgraded Laravel project doesn't render correctly

### DISCUSSION THREAD

**Discussion by LNCH** [livewire/livewire#4468] [1. Help]
I've recently updated an existing project from Laravel 5.x to Laravel 8. After doing this, I've installed Livewire 2.x and attempted to create a fairly simple component.

Livewire for all intents and purposes _seems_ to be working, however it is only rendering the initial opening tag and then ignoring the closing tag and any content found within those tags. 

I'm getting errors such as $slot is undefined, from the Livewire component, because the children of the livewire component tag is not being rendered. 

I'm at a loss now as to what could be missing, but with the original project being upgraded I'm not certain if something might be missing/intefering.

**Comment by joshhanley** [comment]
@LNCH hard to say without seeing the project. Not sure what you mean by the opening tag is working but not the closing tag, can you explain that some more?

If you could share some code, like your route, controller, view, LW component class and view, and also your layout file so we can have a look at it?

Thanks!
